code/WorkInProgress/infernal_contracts.dm

```diff
@@ -726,6 +727,21 @@ obj/item/contract/reversal
 
 		return 1
 
+obj/item/contract/krampus
```

Comment:
not a big fan of krampuses in generally, they're so buggy

Reply:
They're not much different from a regular meatcube anymore, besides the lack of a popping timer and the artificially old age for the scream sound effects.

Unless you're referring to krampus 1.0 (the non-cube krampus), which this contract does **not** make use of.